src/notebook/kernel.ts

```diff
@@ -204,6 +204,17 @@ export class Kernel implements IKernel {
         // name is not needed
       } as unknown as UpdateCellIdsRequest);
     }
+
+    // Update markdown cells to run on change.
+    const markdownChanges = e.cellChanges
+      .flatMap((change) => change.cell)
+      .filter((cell) => cell.document.languageId === MARKDOWN_LANGUAGE_ID);
```

Comment:
i think i tried this at one point - and it was sending off a new request on type. which made the kiosk quite jarring. but i maybe didnt do this exact thing - did you notice anything like that?

Reply:
It does update on keystroke, but seemed more "snappy", than jarring. Instant update and render

Maybe the cell highlights? But I didn't think it was that bad

Reply:
sounds good - i trust you on that. thanks for adding this!